### DISCUSSION TITLE
haproyx-wi install.sh missing

### DISCUSSION THREAD

**Discussion by pmazzolaru** [roxy-wi/roxy-wi#275] [General]
I have an haproxy server built on the latest version of CentOS 8.  I am trying to install haproxy-wi on the haproxy server.  I followed the below steps but it seems there is no install.sh located in the repository and is therefore never cloned to /var/www/haproxy-wi.

yum -y install git wget
git clone https://github.com/Aidaho12/haproxy-wi.git /var/www/haproxy-wi
cd /var/www/haproxy-wi
chmod +x install.sh <-----error ...file not found
./install.sh

**Comment by Aidaho12** [comment]
Yep, it was deleted long time ago...

**Reply by pmazzolaru** [reply]
So how do I get the haproxy-wi installed?

**Comment by Aidaho12** [comment]
Read the installation instruction on the site, I guess

**Reply by pmazzolaru** [reply]
Wow..The installation instructions are so worng.  I perform the below:

**Reply by pmazzolaru** [reply]
yum -y install git wget
git clone https://github.com/Aidaho12/haproxy-wi.git /var/www/haproxy-wi		
yum -y install python36 python3-pip python3-devel python3-cryptography python3-jinja2 nmap-ncat net-tools lshw python3-ldap rsync ansible dos2unix nmap mod_ssl httpd
pip3 install -r haproxy-wi/config_other/requirements_el8.txt
cp haproxy-wi/config_other/httpd/* /etc/httpd/conf.d/
chmod +x haproxy-wi/app/*.py 
cp haproxy-wi/config_other/logrotate/* /etc/logrotate.d/
cp haproxy-wi/config_other/syslog/* /etc/rsyslog.d/
systemctl daemon-reload      
systemctl restart httpd
systemctl restart rsyslog
mkdir /var/www/haproxy-wi/app/certs
mkdir /var/www/haproxy-wi/keys
mkdir /var/www/haproxy-wi/configs/
mkdir /var/www/haproxy-wi/configs/hap_config/
mkdir /var/www/haproxy-wi/configs/kp_config/
mkdir /var/www/haproxy-wi/configs/nginx_config/
mkdir /var/www/haproxy-wi/log/
chown -R apache:apache /var/www/haproxy-wi/

**Reply by pmazzolaru** [reply]
I figured it out...Thanks

**Reply by Aidaho12** [reply]
What there is so wrong?
